File: src/main/java/net/Chidoziealways/everythingjapanese/block/state/properties/ModBlockStateProperties.java
package net.Chidoziealways.everythingjapanese.block.state.properties;

import net.minecraft.world.level.block.state.properties.BooleanProperty;
import net.minecraft.world.level.block.state.properties.IntegerProperty;

public class ModBlockStateProperties {
    public static final IntegerProperty BITES  = IntegerProperty.create("bites", 0, 16);
    public static final BooleanProperty CLICKED = BooleanProperty.create("clicked");
    public static final IntegerProperty AGE_4 = IntegerProperty.create("age", 0, 4);
}

File: src/main/kotlin/net/Chidoziealways/everythingjapanese/block/state/properties/ModBlockStateProperties.kt
package net.Chidoziealways.everythingjapanese.state.properties

import net.minecraft.world.level.block.state.properties.BooleanProperty
import net.minecraft.world.level.block.state.properties.IntegerProperty

object ModBlockStateProperties {
    val BITES_16: IntegerProperty = IntegerProperty.create("bites", 0, 16)
    val BITES_11: IntegerProperty = IntegerProperty.create("bites", 0, 11)
    val CLICKED: BooleanProperty = BooleanProperty.create("clicked")
    val AGE_4: IntegerProperty = IntegerProperty.create("age", 0, 4)
    val OPEN: BooleanProperty = BooleanProperty.create("open")
}
